husky-communication/husky-communication-gen/src/main/java/org/projecthusky/communication/utils/HuskyUtils.java

```diff
@@ -0,0 +1,122 @@
+/**
+ * 
+ */
+package org.projecthusky.communication.utils;
+
+import java.net.URI;
+
+/**
+ * Helper class for husky specific functionality.
+ */
+public abstract class HuskyUtils {
+
+	public static final String SERVER_IN_LOGGER = "#serverInLogger";
+	public static final String SERVER_OUT_LOGGER = "#serverOutLogger";
+	public static final String AUDIT_CONTEXT = "#auditContext";
+	// &httpClientPolicy=#myHttpClientPolicy
+	public static final String HTTP_CLIENT_POLICY = "#huskyHttpClientPolicy";
+	public static final String HTTPS_LITERAL = "https://";
+	public static final String HTTP_LITERAL = "http://";
+
+	public static final String HPD_ITI_58 = "hpd-iti58";
+	public static final String HPD_ITI_59 = "hpd-iti59";
+	public static final String PDQV3_ITI47 = "pdqv3-iti47";
+	public static final String PIXV3_ITI44 = "pixv3-iti44";
+	public static final String PIXV3_ITI45 = "pixv3-iti45";
+	public static final String XDS_ITI18 = "xds-iti18";
+	public static final String XDS_ITI43 = "xds-iti43";
+	public static final String XDS_ITI41 = "xds-iti41";
+	public static final String CH_PPQ1 = "ch-ppq1";
+	public static final String CH_PPQ2 = "ch-ppq2";
+
+//	"xds-iti18://%s?inInterceptors=%s&inFaultInterceptors=%s&outInterceptors=%s&outFaultInterceptors=%s&secure=%s&audit=%s&auditContext=%s&httpClientPolicy=%s"
```

Comment:
delete comment or put it in a context - alternativ, maybe it makes sense to have an endpoint util which knows all the endpoints as constants